customTests/azure_hw_topology_check.nhc

```diff
@@ -40,10 +40,23 @@ function get_actual_topology() {
 }
 
 function check_hw_topology() {
-    TOPO_FILE="$1"
+
+    SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+    SKU="${SKU,,}"
+
+
+    if [[ $SKU == *"nd96"*"_v4"* ]]; then
```

Comment:
Do we need topo files for any of the new skus added?

Reply:
The ncv4, ndv4, ndv5 are the only ones being tested. I think its a good Idea to expand the test to other SKUs but let me plan for this on another work item.